react/next/utils/next.ts

```diff
@@ -0,0 +1,70 @@
+import { SliderState, SliderProps } from '../typings/global'
+
+interface NextSlides {
+  nextSlides: number | undefined
+  nextPosition: number | undefined
+}
+
+/**
+ * Populate the next slides
+ * TODO: Implement the 'fake' translate when in infinite mode when right end is reached
+ * @param state : slider current state
+ * @param props : slider props
+ * @param slidesHavePassed : slides that passed
+ */
+const populateNextSlides = (
+  state: SliderState,
+  props: SliderProps,
```

Comment:
if you refactored the component that uses this function, and lifted its state to a parent component, this function will stop receiving that variable in the first argument and start receiving it in the second. it makes this function more error-prone and fragile

Reply:
I did this to make that function simpler. since just spreading `state` and `props` will return more arguments than expected and causa a ts error. In fact, i do not see a problem in this "awareness". If you could give an example of it beeing a problem i may see it.